spec/factories/state_file_az_intakes.rb

```diff
@@ -232,7 +228,7 @@
     end
 
     factory :state_file_az_refund_intake do
-      after(:build) do |intake, _evaluator|
+      after(:build) do |intake, evaluator|
```

Comment:
was this added by your editor/rubocop? I think we can add it back

Reply:
It was an automatic rubocop change on save. 

> Lint/UnusedBlockArgument: Unused block argument - `evaluator`. If it's necessary, use `_` or `_evaluator` as an argument name to indicate that it won't be used.RuboCop(Lint/UnusedBlockArgument)